packages/openlmis/src/Utils.js

```diff
@@ -0,0 +1,98 @@
+import { composeNextState } from '@openfn/language-common';
+import {
+  request as commonRequest,
+  logResponse,
+  makeBasicAuthHeader,
+} from '@openfn/language-common/util';
+
+export const authorize = state => {
+  const {
+    baseUrl,
+    access_token,
+    username,
+    password,
+    clientId = 'user-client',
+    clientSecrete = 'changeme',
+  } = state.configuration;
+
+  const headers = makeBasicAuthHeader(clientId, clientSecrete);
+
+  if (access_token) return state;
+
+  if (username && password) {
+    if (clientId === 'user-client')
+      console.warn('Using default client id:', clientId);
+    if (clientSecrete === 'changeme')
+      console.warn('Using default client secrete:', clientSecrete);
+
+    const options = {
+      query: { grant_type: 'password', username, password },
+      headers: {
+        'Content-Type': 'application/json',
+        ...headers,
+      },
+      baseUrl,
+      parseAs: 'json',
+      maxRedirections: 1,
+    };
+    return commonRequest('POST', '/api/oauth/token', options)
+      .then(response => {
+        return {
+          ...state,
+          configuration: {
+            ...state.configuration,
+            access_token: response.body.access_token,
+          },
+        };
+      })
+      .catch(err => {
+        console.error('Error authenticating with OpenLMIS');
+        console.log(err.body);
```

Comment:
Correct me if I'm wrong, but on the latest CLI and Worker there should be no reason to intercept the exception here. The error should be nicely logged. In fact, but console.logging it I think you're duplicating the error reporting.

Can you check against the latest version?

Reply:
Your right, i was testing using the old CLI version, I have tested with CLI `1.6.0` and the CLI logs the error nicely. 

I have removed the extra error logging

Reply:
@mtuchi  looking at this now, I still see:

```
console.error('Error authenticating with OpenLMIS');
console.log(err.body);
throw err
```

I don't think we want to log err.body?

Reply:
Yeah your right, i have remove this catch entirely. The error log that i am getting for `401` is more than enough